src/dash-table/components/CellFactory.tsx

```diff
@@ -47,14 +47,14 @@ export default class CellFactory {
             style_cell_conditional,
             style_data,
             style_data_conditional,
-            viewport
+            virtualized
         } = this.props;
 
         const operations = this.cellOperations(
             active_cell,
             data,
-            viewport.data,
-            viewport.indices,
+            virtualized.data,
+            virtualized.indices,
```

Comment:
Viewport is *not* the final result of the table processing its content anymore, now it's Virtualized!

data -> virtual_data -> viewport_data -> virtualized_data

If virtualization=false, virtualized_data === viewport_data + offset of zero